STATUS: INLINE SUCCESSFUL
MODIFIED: de.strullerbaumann.visualee.filter.entity.SourcecodeFilterTest.testIsOkIncludeTrue
MODIFIED FILE: /home/alex/Repositories/intIDRepair/toolRepos/ODRepair/experiments/_downloads/visualee/src/test/java/de/strullerbaumann/visualee/filter/entity/SourcecodeFilterTest.java
CLEANER: de.strullerbaumann.visualee.filter.entity.TestGroup100Case0.test254
CLEANER FILE: /home/alex/Repositories/intIDRepair/toolRepos/ODRepair/experiments/_downloads/visualee/src/test/java/de/strullerbaumann/visualee/filter/entity/TestGroup100Case0.java
POLLUTER: de.strullerbaumann.visualee.filter.entity.SourcecodeFilterTest.testIsOkIncludeFalse
POLLUTER FILE: /home/alex/Repositories/intIDRepair/toolRepos/ODRepair/experiments/_downloads/visualee/src/test/java/de/strullerbaumann/visualee/filter/entity/SourcecodeFilterTest.java
ORIGINAL CLEANER SIZE: 16
NEW CLEANER SIZE: 2
ELAPSED TIME: 108.76699995994568
==========================
@@ -113,0 +113,4 @@
+ {
+     de.strullerbaumann.visualee.filter.boundary.FilterContainer filterContainer0 = de.strullerbaumann.visualee.filter.boundary.FilterContainer.getInstance();
+     filterContainer0.clear();
+ }
